src/components/AirdropWalletsCarousel.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { Dimensions, ViewStyle } from 'react-native';
+import Carousel from 'react-native-snap-carousel';
+
+import { AirdropWalletDetails } from 'app/consts';
+
+import { AirdropWalletsCarouselItem } from './AirdropWalletsCarouselItem';
+
+interface Props {
+  items: AirdropWalletDetails[];
+  styles?: ViewStyle;
+}
+
+const SCREEN_WIDTH = Dimensions.get('window').width;
```

Comment:
Two things:
- As I mentioned in another comment, I believe it'd be better if such variables would be an actual part of a component instead of being defined somewhere next to it. 
- it's totally subjective, but imho `SCREEN_WIDTH` should be written in camelCase since it's not a constant, but a result of the `Dimensions.get('window')` function calculation

Reply:
1. will wait for @KedziaPawel input
2. took it straight from WalletsCarousel component that we already have implemented. Would rather keep it consistent, may change in both places if you wish :)

Reply:
to be honest I also would be kept it outside the component, no need the create it on every rerender

Reply:
maybe you're right, let's leave it as it is.

so, @jellyfish-tom , it's being used in at least two places maybe it'd be a good idea to extract it to some common module, like helpers or something?